symfony/src/Controller/Admin/DashboardController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Controller\Admin;
+
+
+use App\Base\BaseController;
+use App\Manager\StatisticsManager;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/admin/dashboard", name="admin_dashboard_")
+ */
+class DashboardController extends BaseController
```

Comment:
I guess you can rename it Statistics to match the path name

Reply:
ah no ok, you have a dashboard with statistics and potentially something else, allright